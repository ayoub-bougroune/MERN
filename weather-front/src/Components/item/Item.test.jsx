import React from 'react';
import renderer from 'react-test-renderer';
import ItemVilles from './index';
import mockItems from '../../utils/mockData';
import { ThemeProvider } from '@material-ui/styles';
import { createMuiTheme } from '@material-ui/core/styles';
const theme = createMuiTheme({
  palette: {
    primary: { main: "#00FF9B", contrastText: "#fff" },
    secondary: { light : "#8c75e0", main: "#432C85", contrastText: "#fff" },
    info:{light : "#01057a", main: "#701b86", contrastText: "#fff"},
    type: 'dark'
  }
});
test('The ItemVille renders correctly', () => {
  const itemVille = renderer
    .create(<ThemeProvider theme={theme}><ItemVilles villes={mockItems[0]}/></ThemeProvider>);
  const tree = itemVille.toJSON();
  expect(tree).toMatchSnapshot();
});