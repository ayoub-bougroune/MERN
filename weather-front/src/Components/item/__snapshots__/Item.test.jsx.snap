// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The ItemVille renders correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-5 MuiGrid-align-items-xs-center"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-md-4"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-104 MuiPaper-rounded"
    >
      <div
        className="MuiCardContent-root makeStyles-cardTitle-106"
      >
        <p
          className="MuiTypography-root makeStyles-title-105 MuiTypography-body1"
        >
          Casablanca
        </p>
      </div>
      <div
        className="MuiCardMedia-root makeStyles-media-108"
        style={
          Object {
            "backgroundImage": "url(\\"/assets/Rain_dark.png\\")",
          }
        }
        title="Paella dish"
      >
        <div
          className="MuiCardContent-root makeStyles-cardDegree-107"
        >
          <h1>
            17
            °
          </h1>
        </div>
      </div>
      <div
        className="MuiCardActions-root MuiCardActions-spacing"
      >
        <div
          className="MuiGrid-root MuiGrid-container"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
          >
            <div
              className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column"
            >
              <div
                className="MuiGrid-root"
              >
                <div
                  className="makeStyles-arrowDown-174"
                />
              </div>
              <div
                className="MuiGrid-root"
              >
                11
              </div>
              <div
                className="MuiGrid-root"
                color="primary.main"
              >
                <div
                  className="MuiBox-root MuiBox-root-176"
                >
                  Min
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
          />
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
          >
            <div
              className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column"
            >
              <div
                className="MuiGrid-root"
              >
                <div
                  className="makeStyles-arrowUp-173"
                />
              </div>
              <div
                className="MuiGrid-root"
              >
                17
              </div>
              <div
                className="MuiGrid-root"
                color="#f00"
              >
                <div
                  className="MuiBox-root MuiBox-root-177"
                >
                  Max
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-md-4"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-104 MuiPaper-rounded"
    >
      <div
        className="MuiCardContent-root makeStyles-cardTitle-106"
      >
        <p
          className="MuiTypography-root makeStyles-title-105 MuiTypography-body1"
        >
          Oujda
        </p>
      </div>
      <div
        className="MuiCardMedia-root makeStyles-media-108"
        style={
          Object {
            "backgroundImage": "url(\\"/assets/Rain_dark.png\\")",
          }
        }
        title="Paella dish"
      >
        <div
          className="MuiCardContent-root makeStyles-cardDegree-107"
        >
          <h1>
            11
            °
          </h1>
        </div>
      </div>
      <div
        className="MuiCardActions-root MuiCardActions-spacing"
      >
        <div
          className="MuiGrid-root MuiGrid-container"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
          >
            <div
              className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column"
            >
              <div
                className="MuiGrid-root"
              >
                <div
                  className="makeStyles-arrowDown-174"
                />
              </div>
              <div
                className="MuiGrid-root"
              >
                4
              </div>
              <div
                className="MuiGrid-root"
                color="primary.main"
              >
                <div
                  className="MuiBox-root MuiBox-root-178"
                >
                  Min
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8"
          />
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2"
          >
            <div
              className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column"
            >
              <div
                className="MuiGrid-root"
              >
                <div
                  className="makeStyles-arrowUp-173"
                />
              </div>
              <div
                className="MuiGrid-root"
              >
                11
              </div>
              <div
                className="MuiGrid-root"
                color="#f00"
              >
                <div
                  className="MuiBox-root MuiBox-root-179"
                >
                  Max
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
